pkg/util/certificates.go

```diff
@@ -29,7 +29,7 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-const customCAPath = "/custom/ca"
+const customCAPath = "/tmp/custom/ca"
```

Comment:
Revert this change please

Reply:
will do, it was just for `make test` to work locally, didnt mean to push!

Reply:
done